src/controllers/controller.cpp

```diff
@@ -37,6 +37,24 @@ ControllerJSProxy* Controller::jsProxy() {
     return new ControllerJSProxy(this);
 }
 
+QString Controller::physicalTransport2String(PhysicalTransportProtocol protocol) {
+    switch (protocol) {
+    case PhysicalTransportProtocol::USB:
+        return QStringLiteral("USB");
+    case PhysicalTransportProtocol::BlueTooth:
+        return QStringLiteral("Bluetooth");
+    case PhysicalTransportProtocol::I2C:
+        return QStringLiteral("I2C");
+    case PhysicalTransportProtocol::SPI:
+        return QStringLiteral("SPI");
+    case PhysicalTransportProtocol::FireWire:
+        return QStringLiteral("Firewire - IEEE 1394");
+    case PhysicalTransportProtocol::UNKNOWN:
+    default:
+        return tr("Unknown");
+    }
```

Comment:
nit: remove `default` case and benefit from `-Wswitch`.
```suggestion
    case PhysicalTransportProtocol::UNKNOWN:
        break; // effectively fallthrough
    }
    return tr("Unknown");
```